src/Encoding/JoseEncoder.php

```diff
@@ -23,10 +24,14 @@ final class JoseEncoder implements Encoder, Decoder
     public function jsonEncode(mixed $data): string
     {
         try {
-            return json_encode($data, JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE | JSON_THROW_ON_ERROR);
+            $jsonEncoded = json_encode($data, JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE | JSON_THROW_ON_ERROR);
         } catch (JsonException $exception) {
             throw CannotEncodeContent::jsonIssues($exception);
         }
+
+        assert($jsonEncoded !== '');
```

Comment:
Fairly sure this will become redundant once upstream stubs are adjusted (in future). JSON encoding values **never** produces empty strings :shrug:

Reply:
I think this is the kind of thing that I'd just throw a `@phpstan-ignore-line` at instead of adding the assertion.